src/devices/GrovePi/Models/GrovePiCommands.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
```

Comment:
copyright header

Reply:
same comment for other files where it is missing

Reply:
pfff, totally forgot them… Will add